api/v1/postgres_types.go

```diff
@@ -280,6 +283,42 @@ func (p *Postgres) ToPeripheralResourceName() string {
 	return p.generateTeamID() + "-" + p.generateDatabaseName()
 }
 
+// ToUserPasswordSecret returns the secret containing user password pairs
+func (p *Postgres) ToUserPasswordsSecret(src *corev1.SecretList, scheme *runtime.Scheme) (*corev1.Secret, error) {
+	secret := &corev1.Secret{}
+	secret.Namespace = p.Namespace
+	// todo: Consider `p.Name + "-passwords", so the`
```

Comment:
You're right, I think `p.Name` would be better here: We create the secret in the control plane cluster, so we should use the name of the resource in the control plane cluster, not the name iof the resource n the service cluster.

Reply:
cloud-api must be able to detect the name of the secret, currently the backup-secret is stored under:
uuid-backup
Where uuid is the full blown uuid / name of the postgres instance

Reply:
`secretName` is `postgresName-passowrds` now.